applicationconfig/client/src/samples/java/ConfigurationSets.java

```diff
@@ -0,0 +1,105 @@
+import com.azure.applicationconfig.ConfigurationClient;
+import com.azure.applicationconfig.ConfigurationClientCredentials;
+import com.azure.applicationconfig.models.ConfigurationSetting;
+import com.azure.applicationconfig.models.SettingSelector;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import models.KeyVaultConfiguration;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+/**
+ * Sample demonstrates how to use Azure Application Configuration to switch between "beta" and "production"
+ * configuration sets.
+ *
+ * <p>
+ * In the sample, the user stores a connection string to their storage resource and Key Vault connection information.
+ * The Key Vault connection information is serialized into a JSON string and read out from the service as a
+ * strongly-typed typed {@link KeyVaultConfiguration} object.
+ */
+public class ConfigurationSets {
+    private static final String CONNECTION_STRING_KEY = "connection-string";
+    private static final String KEY_VAULT_KEY = "key-vault";
+    private static final String BETA = "beta";
+    private static final String PRODUCTION = "production";
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    public static void main(String[] args) throws NoSuchAlgorithmException, InvalidKeyException, IOException {
+        // Retrieve the connection string from the configuration store.
```

Comment:
I don't understand what this comment is stating. It kind of makes me feel like you're doing something with the configuration service, but clearly you aren't.